api/contract.go

```diff
@@ -95,6 +95,17 @@ func (r *updateFilterOutputRequest) Valid() error {
 	return nil
 }
 
+// getFilterOutputResponse is the response body for GET/filter-outputs/{id}
+type getFilterOutputResponse struct {
+	FilterID         string                  `json:"filter_id"`
+	InstanceID       string                  `json:"instance_id"`
+	DimensionListURL string                  `json:"dimension_list_url"`
```

Comment:
Pretty sure dimensionListURL is no longer part of the spec.. I see the new swagger spec hasn't been approved and merged yet.. probably a good idea to do that

Reply:
New swagger spec has been merged